generators/base-entity-changes/generator.ts

```diff
@@ -176,7 +205,17 @@ export default abstract class GeneratorBaseEntityChanges extends GeneratorBaseAp
         addedRelationships,
         removedRelationships,
         relationshipsToRecreateForeignKeysOnly,
+        removedDefaultValueFields,
+        addedDefaultValueFields,
       };
     });
   }
+
+  private hasAnyDefaultValue(field) {
+    return field.defaultValue || field.defaultValueComputed;
```

Comment:
Falsy value?
```suggestion
    return field.defaultValue !== undefined || field.defaultValueComputed;
```

this function can be used instead of `fieldDefaultValueDefined`?

Reply:
Changed. Thanks for the remark! `fieldDefaultValueDefined` means there is an actual "value" defined (not computed), while this one here is about both types of default values